client/data/courses/constants.ts

```diff
@@ -16,4 +17,14 @@ export const COURSE_DETAILS = {
 			i18n.translate( 'Set yourself up for blogging success' ),
 		],
 	},
+	[ COURSE_SLUGS.PAYMENTS_FEATURES ]: {
+		headerTitle: i18n.translate( 'Add Payments Features' ),
+		headerSubtitle: i18n.translate( 'Make Money on Your Website' ),
+		headerSummary: [
+			i18n.translate( 'Accept one-time or recurring payments' ),
+			i18n.translate( 'Accept donations or sell services' ),
+			i18n.translate( 'Setup paid, subscriber-only content' ),
+			i18n.translate( 'Run a fully featured ecommerce store' ),
```

Comment:
Speaking of `ecommerce`, I've noticed that we write it as `eCommerce` in the video texts.

I've double-checked with the Editorial team, and the preferred format is `ecommerce` (or `Ecommerce` when appropriate, e.g. in a title, or after a period).

Would you mind updating the video titles and descriptions by replacing `eCommerce` with `ecommerce`?

Reply:
Updated.